### PR TITLE
Ras 306

### PR DESCRIPTION
**Pull request by zefciu**

### CODE REVIEW

File: src/ralph_scrooge/rest/pricing_objects.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+from django.conf import settings
+from django.db.models import Sum
+from django.template.defaultfilters import slugify
+
+from ralph_scrooge.rest.components import ComponentsContent
+from ralph_scrooge.models import DailyCost, PricingObject
+
+
+class PricingObjectsContent(ComponentsContent):
```
Comment (mkurek) [inline src/ralph_scrooge/rest/pricing_objects.py]: rename it to sth related with cost, ex. `PricingObjectsCostContent`

File: src/ralph_scrooge/rest/pricing_objects.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            daily_pricing_objects__date__gte=start_date,
+            daily_pricing_objects__date__lte=end_date,
+        )
+        if env:
+            query = query.filter(
+                daily_pricing_objects__service_environment__environment__id=env
+            )
+        pricing_object_ids = query.values_list('id', flat=True)
+        query_daily_cost = DailyCost.objects.filter(
+            pricing_object_id__in=pricing_object_ids,
```
Comment (kula1922) [inline src/ralph_scrooge/rest/pricing_objects.py]: why just not 

pricing_object_id__in=query.values_list('id', flat=True)

File: src/ralph_scrooge/rest/pricing_objects.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            date__gte=start_date,
+            date__lte=end_date,
+            service_environment__service__id=service
```
Comment (mkurek) [inline src/ralph_scrooge/rest/pricing_objects.py]: add filter to get only costs for accepted dates, ex. 
```
date__in=CostDateStatus.objects.filter(accepted=True, date__gte=start_date, date__lte=end_date).values_list('date', flat=True)
```

File: src/ralph_scrooge/rest/pricing_objects.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+        )
+        if env:
+            query_daily_cost = query_daily_cost.filter(
+                service_environment__environment__id=env
```
Comment (kula1922) [inline src/ralph_scrooge/rest/pricing_objects.py]: service_environment__environment_id=env for optimalization

File: src/ralph_scrooge/rest/urls.py [hunk 5]
```diff
@@ -46,10 +48,18 @@
...
     url(
+        r'^components/(?P<service>\d+)/(?P<env>\d+)/(?P<year>\d+)/(?P<month>\d+)/(?P<day>\d+)/?$',  # noqa
+        service_permission(ComponentsContent.as_view()),
+    ),
+    url(
         r'^costcard/(?P<service>\d+)/(?P<env>\d+)/(?P<year>\d+)/(?P<month>\d+)/?$',  # noqa
         service_permission(CostCardContent.as_view()),
     ),
     url(
+        r'^pricing_objects/(?P<service>\d+)/(?P<env>\d+)/(?P<start_date>[0-9-]+)/(?P<end_date>[0-9-]+)/?$',  # noqa
```
Comment (mkurek) [inline src/ralph_scrooge/rest/urls.py]: rename to `pricing_objects_costs`

File: src/ralph_scrooge/rest/pricing_objects.py (hunk 3 above)
Comment (kula1922) [inline src/ralph_scrooge/rest/pricing_objects.py]: dito

File: src/ralph_scrooge/settings.py [hunk 6]
```diff
@@ -49,6 +49,41 @@
 
 TESTING = 'test' in sys.argv
 
+PRICING_OBJECTS_TABLE_SCHEMA = {
```
Comment (mkurek) [inline src/ralph_scrooge/settings.py]: `PRICING_OBJECTS_COSTS_TABLE_SCHEMA`

**Comment by ar4s** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

**Pull request merged**